opensrp-tasking/src/main/java/org/smartregister/tasking/contract/TaskingHomeActivityContract.java

```diff
@@ -0,0 +1,169 @@
+package org.smartregister.tasking.contract;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.StringRes;
+
+import com.mapbox.geojson.Feature;
+import com.mapbox.geojson.FeatureCollection;
+import com.mapbox.mapboxsdk.geometry.LatLng;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.smartregister.commonregistry.CommonPersonObjectClient;
+import org.smartregister.domain.Event;
+import org.smartregister.domain.Task;
+import org.smartregister.tasking.model.CardDetails;
+import org.smartregister.tasking.model.TaskDetails;
+import org.smartregister.tasking.model.TaskFilterParams;
+import org.smartregister.tasking.util.TaskingJsonFormUtils;
+
+import java.util.List;
+
+/**
+ * Created by samuelgithengi on 11/27/18.
+ */
+public interface TaskingHomeActivityContract {
```

Comment:
The name of this contract does not easily describe what is does/provides

Reply:
updated